packages/oas-normalize/src/index.ts

```diff
@@ -224,8 +224,18 @@ export default class OASNormalize {
        * @deprecated
        */
       parser?: ParserOptions;
+
+      /**
+       * Should `.validate()` throw a `ValidationError` if the API definition is invalid?
```

Comment:
```suggestion
       * Determines if `.validate()` throws a `ValidationError` if the API definition is invalid
```
Nit: can we word this less like a question? I got a little confused reading this comment before realizing this was a boolean prop

Reply:
sure